spec/features/account/backup_codes_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+feature 'Backup codes' do
```

Comment:
A thing we may want to consider adding to this spec is something that clicks the button to generate codes an verifies that they are generated.